src/Fibonacci.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class Fibonacci {
+
+	public static void main(String[] args) {
+		int n, first = 0, next = 1;
+		System.out.println(" Enter the value of n : ");
+		Scanner scanner = new Scanner(System.in);
+		n = scanner.nextInt();
+		System.out.print(" Fibonacci Series is : ");
+		System.out.print(first + " " + next);
+		for (int i = 1; i <= n - 2; i++) {
```

Comment:
As per requirement, our output should start from 1 and end on 55 when user enters 10
your logic is correct, just not matching with the expected output.

Reply:
Changes Incorporated.

Reply:
![image](https://user-images.githubusercontent.com/66831155/86121992-9f8fa800-baf4-11ea-9672-d2e02a48a0d8.png)